src/packages/cli/src/__tests__/integration/mssql/__database.ts

```diff
@@ -1,34 +1,45 @@
-const sql = require('mssql')
+import mssql from 'mssql'
+import url from 'url'
+
 import { Context, Input } from '../../__helpers__/integrationTest'
 
 export const database = {
-  name: 'mssql',
+  name: 'sqlserver',
   datasource: {
-    url: (ctx) => getConnectionInfo(ctx).connectionString,
+    url: ctx =>
+      `sqlserver://localhost:1433;database=master_tests;user=SA;password=Prisma1-prisma;trustServerCertificate=true;encrypt=DANGER_PLAINTEXT`,
   },
   connect(ctx) {
     const credentials = getConnectionInfo(ctx).credentials
-    return sql.connect(credentials)
+    return mssql.connect({
+      user: credentials.user,
+      password: credentials.password,
+      server: credentials.server,
+      database: ctx.step === 'scenario' ? `master_tests` : `master`,
+    })
   },
   send: (db, sql) => db.query(sql),
-  close: (db) => db.end(),
-  up: (ctx) => {
-    return `
-        DROP DATABASE IF EXISTS ${ctx.id};
-        CREATE DATABASE ${ctx.id};
-        USE ${ctx.id};
-        `
-  },
+  close: db => db.close(),
+  up: ctx =>
+    `
+        USE master;
+        DROP DATABASE IF EXISTS "master_tests";
+        CREATE DATABASE "master_tests";
```

Comment:
I've changed it so the database is the same for all tests, rather than have it be derived from the context. Should have the same ultimate effect.

Reply:
Why not have them be dynamically created based on context? I used the logic of you added for the step, but switched the database name to be dynamically created each time i.e.`master_<ctx.id>`
Should be visible in the latest commit: https://github.com/prisma/prisma/compare/feat/integration-tests-mssql
I also noticed the teardown function will probably need to be adapted to work as the test databases aren't disappearing. But you think we should hard code one test database?

Reply:
Oh yes, you're right, we can totally create them dynamically. I didn't realize that I'm connecting to a different database during the database setup and the scenario setup, so there shouldn't be problems.

I also didn't look at the teardown, so you're correct there too.